Misc/NEWS.d/next/Library/2023-06-12-10-40-38.gh-issue-105684.yiHkFD.rst

```diff
@@ -0,0 +1 @@
+:meth:`asyncio.Task.set_name` is now mandatory for third party task implementations. Patch by Kumar Aditya.
```

Comment:
```suggestion
Supporting :meth:`asyncio.Task.set_name` is now mandatory for third party task implementations.
The undocumented `_set_task_name` function (deprecated since 3.8) has been removed.
Patch by Kumar Aditya.
```